elephant/spike_train_dissimilarity.py

```diff
@@ -363,6 +363,18 @@ def van_rossum_distance(spiketrains, time_constant=1.0 * pq.s, sort=True):
     for i, j in np.ndindex(k_dist.shape):
         vr_dist[i, j] = (
             k_dist[i, i] + k_dist[j, j] - k_dist[i, j] - k_dist[j, i])
+
+    # Clip small negative values
+    if np.any(vr_dist < 0):
+        warnings.warn(
+            "van_rossum_distance: very small negative values encountered; "
+            "setting them to zero. Potentially due to floating point error, "
+            "which can occur if spike times are represented as small floating "
+            "point values (e.g., in seconds). A possible way to prevent this "
+            "warning is to use a time unit with better numerical precision, "
+            "e.g. seconds to milliseconds.", RuntimeWarning)
```

Comment:
The revised warning looks good. I have a small suggestion for improved grammar `e.g., from seconds` instead of `e.g. seconds`